components/renku_data_services/project/blueprints.py

```diff
@@ -112,29 +101,17 @@ def get_one(self) -> BlueprintFactoryResponse:
         @authenticate(self.authenticator)
         @validate_path_project_id
         async def _get_one(request: Request, user: base_models.APIUser, project_id: str) -> JSONResponse | HTTPResponse:
-            project = await self.project_repo.get_project(user=user, project_id=ULID.from_str(project_id))
+            with_documentation = bool(request.args.get("with_documentation", False))
```

Comment:
issue: use query string validation instead with the apispec model (same as here: https://github.com/SwissDataScienceCenter/renku-data-services/blob/5b095d795f96247aa559a3083947283c534c355d/components/renku_data_services/project/blueprints.py#L43 )